wgpu-hal/src/gles/egl.rs

```diff
@@ -840,6 +840,8 @@ impl Surface {
                 crate::SurfaceError::Lost
             })?;
 
+        gl.disable(glow::SCISSOR_TEST);
```

Comment:
let's also reset the color mask, which could show the same issue

Reply:
Added!